### PR TITLE
feature/ST 157 https server

### PR DESCRIPTION
**Pull request by james-simpson-ST**
This is a working https server but I have not been able to test with an Apple device as I am not working from home today. I have got it to the point where there is no errors in chrome when calling ApplePay but that is as far as I can test

### CODE REVIEW

**Comment by james-simpson-ST** [comment]
N.B. I will also be updating the test framework to use https on the other repo

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "webpack --config webpack.prod.js",
     "start": "webpack-dev-server --color --progress --config webpack.dev.js",
-    "example": "live-server ./dist --port=9090 --host=localhost --no-browser",
+    "example": "ws --rewrite '/* -> /dist/$1' --https",
```
Comment (pgs-gmartowski) [inline package.json]: how should I start application now to have https available ? I start only `example` profile and has some npm errors

**Comment by james-simpson-ST** [comment]
The build is failing for the same tests as the apple pay branch and so we can pull this into the apple pay branch in it's current state of not passing the build

File: webpack.dev.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ module.exports = merge(common, {
   devtool: 'inline-source-map',
   devServer: {
     contentBase: './dist',
-    port: 8080,
+    port: 8443,
```
Comment (pgs-gmartowski) [inline webpack.dev.js]: why exactly this port number ?

File: package.json (hunk 1 above)
Comment (james-simpson-ST) [reply package.json]: Can you send me the traceback

You should also be able to run npm start as it should be able to run using the dev server still

File: webpack.dev.js (hunk 2 above)
Comment (james-simpson-ST) [reply webpack.dev.js]: On linux 443 is a protected port which only root can start, 8443 is commonly used for testing https via a port that doesn't need root to start the server

Comment (pgs-gmartowski) [reply webpack.dev.js]: ok, thx :)

File: package.json (hunk 1 above)
Comment (pgs-gmartowski) [reply package.json]: that's weird, now it's working:/ maybe I've got some problems with npm (I'm switching branches all the time).

**Comment by pgs-gmartowski** [comment]
@james-simpson-ST  approved

File: package.json (hunk 1 above)
Comment (aaron-moore-ST) [reply package.json]: I have had some trouble with needing an npm install after switching branches if they have different packages being added, that could be what you ran into.

Comment (pgs-gmartowski) [reply package.json]: yup, I think that could be the reason

**Pull request merged**